hooks/queries/governancePower.ts

```diff
@@ -102,6 +106,20 @@ export const getNftGovpower = async (
   return power
 }
 
+export const getPythGovPower = async (connection: Connection, user : PublicKey | undefined) : Promise<BN> => {
+  if (!user) return new BN(0)
+
+  const pythClient = await PythClient.connect(connection, new NodeWallet(new Keypair()))
```

Comment:
This client is readonly and I don't know any more elegant way of creating a "random wallet"

Reply:
Yeah I mean it seems like a wallet should not actually be required by your SDK. But this seems sufficiently elegant as a way to pass a dummy wallet. You can also just like pass `{} as NodeWallet`.